### ISSUE TITLE
Single copy of coherence controllers

### ISSUE THREAD

**Issue by smpallen99** [smpallen99/coherence#36]
There are currently two copies of each controller, one under `web/controllers` and the other as a template under priv. The only difference between the two copies is the the first line where the template version has an eex statement to add the namespace for the generator. 

To reduce the copy and pasting during dev, I would like to remove the template version and modify the installer to just read the controllers from `web/controllers` and insert the namespace before writing it to the project's `web/controllers/coherence` directory.

**Comment by kimquy** [comment]
Yeah. It gonna be way better!

**Comment by smpallen99** [comment]
I looked into this and ran into an issue. Mix generators that use templates take them from the referring projects _build/env/lib/package/priv folder which is part of the dependency installation. However, the controllers in located in deps/coherence/web/controllers which is difficult to find (is not located in deps/...  when path: "..." is used.

So, I ended up create a helper script that copies the controllers from the the coherence/web/controllers directory, adds the template code and places them in the templates directory. 

I figure this is the next best solution.
